lib/daru/dataframe.rb

```diff
@@ -545,6 +545,17 @@ def add_vector n, vector
       self[n] = vector
     end
 
+    def insert_vector n, name, source
+      raise ArgumentError unless source.is_a? Array
+      vector = Daru::Vector.new(source, index: @index, name: @name)
```

Comment:
Can we have choice to pass vector directly to be inserted (think about other ways as well)?